library/include/rocwmma/internal/swizzle.hpp

```diff
@@ -70,12 +70,36 @@ namespace rocwmma
         struct Driver
         {
         private:
+            template <typename DataT,
+                      std::enable_if_t<sizeof(DataT) == sizeof(uint64_t), bool> = true>
+            ROCWMMA_DEVICE static inline auto swizzle(DataT&& v)
+            {
+                constexpr uint32_t B32VecSize = 2;
+                using B32VecT                 = VecT<uint32_t, B32VecSize>;
+
```

Comment:
same comment as DPP

Reply:
fixed in commit - " Use vector_generator to replace forEach"